charts/arbitrum-nitro/values.yaml

```diff
@@ -68,8 +68,38 @@ grafana:
   dashboardsConfigMapLabelValue: "1"
 
 nitro:
-  # -- Additional CLI arguments to pass to `nitro`
-  extraArgs: []
+  # -- Nitro configuration parameters
+  config:
+    # -- Chain ID, 42161 for Arbitrum One
+    chain: 42161
+    # -- RPC URL to L1 chain (ethereum)
+    parentChainURL: "CHANGE_ME_RPC_URL"
+    # -- RPC Url to Arbitrum Classic Archive node if serving classic blocks
+    classicURL: null
+    # -- RPC config parameters
+    httpRPC:
+      # -- Enabled APIs
+      api: net,web3,eth,debug
+      # -- Listen address
+      addr: 0.0.0.0
+      # -- Allowed vhosts
+      vhosts: "*"
+      # -- Allowed CORS domains
+      cors: "*"
+    # -- Metrics parameters
+    metrics:
+      # -- Enable metrics
+      enabled: true
+      # -- Listen address
+      addr: 0.0.0.0
+
+    # -- Enabled default arguments on the chart
+    defaultArgs:
+      - "--node.caching.archive"
```

Comment:
this option changes to `--execution.caching.archive` in v2.2.0 are we sure we want to include it as default?

Reply:
> this option changes to `--execution.caching.archive` in v2.2.0 are we sure we want to include it as default?

yes, our defaults are supposed to work for an archive node and that option is needed for the image version this chart comes with. When we update it, for a new chart version, we'll need to change the flag to the new one.